.gitignore

```diff
@@ -4,3 +4,4 @@
 /doc
 erl_crash.dump
 *.ez
+*.ex~
```

Comment:
This should be set in your global ignore and not per project (since this is editor specific - otherwise each .gitignore on all projects would have to have this setting and many others for different editors).

Reply:
good idea, will do.

Reply:
I missed that my local gitignore was being pushed,  Dang.